lib/net.js

```diff
@@ -1398,6 +1415,7 @@ Server.prototype.listen = function(...args) {
     listenInCluster(this, null, -1, -1, backlogFromArgs);
     return this;
   }
+  addAbortSignalOption(this, options);
```

Comment:
If this function is only used once, is it relevant to have it as a separate function instead of inlining it?

Reply:
I just thought that it would make the `listen` function harder to read.